sdk/keyvault/Azure.Security.KeyVault.Administration/tests/samples/Sample1_RbacHelloWorld.cs

```diff
@@ -0,0 +1,156 @@
+        [SyncOnly]
+        public void CreateClient()
+        {
+            // Environment variable with the Key Vault endpoint.
+            string keyVaultUrl = TestEnvironment.KeyVaultUrl;
+
+            #region Snippet:HelloCreateKeyVaultAccessControlClient
+            KeyVaultAccessControlClient client = new KeyVaultAccessControlClient(new Uri(keyVaultUrl), new DefaultAzureCredential());
+            #endregion
+            client = Client;
+        }
+
+        [Test]
+        [SyncOnly]
+        public void GetDefinitionsAndAssignmentsSync()
+        {
+            // Replace client with the Instrumented Client.
+            client = Client;
+
+            // Retrieve all the role definitions.
+            #region Snippet:GetRoleDefinitionsSync
+            List<RoleDefinition> roleDefinitions = client.GetRoleDefinitions(RoleAssignmentScope.Global).ToList();
+            #endregion
+
+            // Retrieve all the role assignments.
+            #region Snippet:GetRoleAssignmentsSync
+            List<RoleAssignment> roleAssignments = client.GetRoleAssignments(RoleAssignmentScope.Global).ToList();
+            #endregion
+
+        }
+
+        [Test]
+        [AsyncOnly]
+        public async Task GetDefinitionsAndAssignmentsAsync()
+        {
+            // Replace client with the Instrumented Client.
+            client = Client;
+
+            // Retrieve all the role definitions.
+            #region Snippet:GetRoleDefinitionsAsync
+            List<RoleDefinition> roleDefinitions = new List<RoleDefinition>();
+            await foreach (var definition in client.GetRoleDefinitionsAsync(RoleAssignmentScope.Global))
+            {
+                roleDefinitions.Add(definition);
+            }
+            #endregion
+
+            // Retrieve all the role assignments.
+            #region Snippet:GetRoleAssignmentsAsync
+            List<RoleAssignment> roleAssignments = new List<RoleAssignment>();
+            await foreach (var assignment in client.GetRoleAssignmentsAsync(RoleAssignmentScope.Global))
+            {
+                roleAssignments.Add(assignment);
+            }
+            #endregion
+        }
+
+        [Test]
+        [SyncOnly]
+        public void CreateRoleAssignment()
+        {
+            // Replace client with the Instrumented Client.
+            client = Client;
+
+            List<RoleDefinition> definitions = client.GetRoleDefinitions(RoleAssignmentScope.Global).ToList();
+            roleDefinitionId = definitions.FirstOrDefault(d => d.RoleName == roleName).Id;
+
+            // Replace roleDefinitionId with a role definition Id from the definitions returned from GetRoleAssignments.
+            string definitionIdToAssign = roleDefinitionId;
+
+            // Replace objectId with the service principal object id.
+            string servicePrincipalObjectId = objectId;
+
+            #region Snippet:CreateRoleAssignment
+            //@@string definitionIdToAssign = <roleDefinitionId>;
```

Comment:
Put quotes around these values, which is 1) consistent with other docs I've seen in our repo, and 2) won't mess up formatting when pasting into VS.